asf/sam/include/sam3u/README

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2023 Chen Xingyu <hi@xingrz.me>
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
README file does not need a copyright notice.

Reply:
Okay. I have no idea. I just follow what c1c4410389eb51ab43a0da271812f3c5328fde17 is did.